pengwin-setup.d/common.sh

```diff
@@ -52,6 +52,21 @@ echo "Removing unnecessary packages."
 sudo apt-get autoremove -y
 }
 
+function command_check() {
+# Usage: command_check <EXPECTED PATH> <ARGS (if any)>
+local execname=$(echo "$1" | sed -e "s|^.*\/||g")
+if "$execname" "$2" > /dev/null 2>&1 ; then
```

Comment:
Please put it in parenthesis for improving the reading

Reply:
done :+1: